hopsworks-common/src/main/java/io/hops/hopsworks/common/project/ProjectAdminInfoDTO.java

```diff
@@ -0,0 +1,89 @@
+package io.hops.hopsworks.common.project;
```

Comment:
We are slowly moving the "data-transfer objects" into the api-layer and out of common. It would be preferable if you did the same with your DTOs. It makes a cleaner separation between logic and api.